app/views/comments/_edit.html.erb

```diff
@@ -1,146 +1,152 @@
-<div class="comment-form-wrapper">
-  <div class="card card-body bg-light">
-    <form method="post" id="c<%= comment.id %>edit" style="display:none;" class="well" <% if comment.is_a?Answer %> action= "/answers/update/<%= comment.id %>" <% else %> action="/comment/update/<%= comment.id %>" <% end %>>
+<div id="edit-comment-form-wrapper-<%= comment.id %>" class="card card-body bg-light comment-form-wrapper" style="display: none;">
+  <form method="post" id="c<%= comment.id %>edit" style="display:none;" class="well" <% if comment.is_a?Answer %> action= "/answers/update/<%= comment.id %>" <% else %> action="/comment/update/<%= comment.id %>" <% end %>>
```

Comment:
This is the fix for the weird superfluous grey bar mentioned [here](https://github.com/publiclab/plots2/issues/8775#issuecomment-762424107). The edit comment form, when opened, was only toggling on `<form>`, not `<form>`'s container.

**Before:** (grey bar below comment)
<img width="749" alt="Screen Shot 2021-01-25 at 9 16 25 PM" src="https://user-images.githubusercontent.com/4361605/105803593-abd8c800-5f52-11eb-94ec-6cf42cff4c57.png">

**After:** (no grey bar)
<img width="717" alt="Screen Shot 2021-01-25 at 9 17 51 PM" src="https://user-images.githubusercontent.com/4361605/105803665-cc088700-5f52-11eb-84c5-89d5ae33aecf.png">